dotCMS/src/main/java/com/dotmarketing/portlets/htmlpageasset/business/render/page/PageView.java

```diff
@@ -76,6 +79,43 @@
         this.pageUrlMapper = pageUrlMapper;
         this.numberContents = getContentsNumber();
         this.live = live;
+        this.urlContent = null;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assigning an Object to null is a code smell.  Consider refactoring.](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4600987)